scripts/dump_nat_entries.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+
+""""
+Description: dump_nat_entries.py -- dump conntrack nat entries from kernel into a file
+             so as to restore them during warm reboot
+"""
+
+import sys
+import subprocess
+
+def main():
+    ctdumpcmd = 'conntrack -L -j > /host/warmboot/nat/nat_entries.dump'
```

Comment:
'conntrack -L -j' output may have a lot of entries. I'm looking at https://github.com/Azure/sonic-buildimage/pull/3494/files#diff-dc200de8bc4641995c83a89e61e3c32cR51 where it is parsed using a complex regular expressions during warm start.
My concern is that parsing that output during system start can consume a lot of cpu in case of huge amount of entries. Did you test this part and impact on system warm reboot?
Right now sonic seems to have performance degradation comparing to 201811 release during system warm start causing LAG/BGP flapping on some platforms.
IMO, dumping this output in a fast-loadable format (like json) during warm start is preferred.

Reply:
This option can be explored. We can do it in a separate PR ?

Reply:
Ok